feature/src/main/feature/feature.xml

```diff
@@ -596,8 +597,9 @@ limitations under the License.
         <bundle>mvn:org.apache.jclouds.karaf.chef/commands/${project.version}</bundle>
     </feature>
 
-     <feature name="jclouds-url-handler" description="Url Handler for jclouds Blobs" version="${project.version}">
+    <feature name="jclouds-url-handler" description="Url Handler for jclouds Blobs" version="${project.version}">
         <feature version='${project.version}'>jclouds</feature>
+        <feature version='${project.version}'>jclouds-services</feature>
```

Comment:
The url handler actually requires the `org.jclouds.blobstore.BlobStore` OSGi service which is provided by the `jclouds-services` feature, so it should be required.
However, the build still succeeds if I remove the above line, so I'll investigate.